src/Tasks/Microsoft.NET.Build.Tasks/GenerateRegFreeComManifest.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+using System.Text;
+using Microsoft.Build.Framework;
+
+namespace Microsoft.NET.Build.Tasks
+{
+    public class GenerateRegFreeComManifest : TaskBase
+    {
+        [Required]
+        public string IntermediateAssembly { get; set; }
+
```

Comment:
What does Intermediate mean in this context?  Is this phrase exposed externally?

Reply:
"Intermediate" is the MSBuild term for the `obj/$Configuration/$TargetFramework/` directory. The `IntermediateAssembly` is the assembly directly output by `Csc` into the `obj` directory. It is later copied to the output directory. This terminology is used throughout MSBuild and the SDK.